connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/streaming/ClientStreamingQuerySuite.scala

```diff
@@ -175,6 +175,38 @@ class ClientStreamingQuerySuite extends QueryTest with SQLHelper with Logging {
     }
   }
 
+  test("throw exception in streaming") {
+    withSQLConf("spark.sql.spark.enrichError.enabled" -> "true") {
+      val session = spark
+      import session.implicits._
+
+      val checkForTwo = udf((value: Int) => {
+        if (value == 2) {
+          throw new RuntimeException("Number 2 encountered!")
+        }
+        value
+      })
+
+      val query = spark.readStream
+        .format("rate")
+        .option("rowsPerSecond", "1")
+        .load()
+        .select(checkForTwo($"value").as("checkedValue"))
+        .writeStream
+        .outputMode("append")
+        .format("console")
+        .start()
+
+      val exception = intercept[SparkException] {
```

Comment:
Shouldn't this be a StreamingQueryException?

Reply:
StreamingQueryException requires some refactoring since spark and connect use different StreamingQueryException at the moment. Will do this in a follow up